common/common_test.go

```diff
@@ -950,3 +953,71 @@ func TestTimeFromUnixTimestampFloat(t *testing.T) {
 		t.Error("Test failed. Common TimeFromUnixTimestampFloat. Converted invalid syntax.")
 	}
 }
+
+func TestGetDefaultDataDir(t *testing.T) {
+	expectedOutput := os.Getenv("APPDATA") + GetOSPathSlash() + "GoCryptoTrader"
+	actualOutput := GetDefaultDataDir("windows")
+	t.Log(actualOutput)
```

Comment:
Can remove this non-needed t.Log and need to add support for getting the default data dir on non-windows based environments.